package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@tacc/core-cms-resources",
+  "version": "3.14.0",
+  "license": "MIT",
+  "description": "Any custom code for all new and updated TACC CMS sites.",
+  "scripts": {
+    "build": "npm run build:css --project=$npm_config_project",
+    "build:css": "_shared/bin/build-css-via-cli.js",
+    "update-version": "_shared/bin/update-version.js"
+  },
+  "// scripts": {
+    "build": "Run all build commands for core and given project",
+    "build:help": "The `$npm_config_` allows `npm run build --project=...`",
+    "build:css": "Build stylesheets from source files for any project"
+  },
+  "repository": "git@github.com:TACC/Core-CMS-Resources.git",
+  "devDependencies": {
+    "yarn": "^1.22.17"
+  },
+  "dependencies": {
+    "@tacc/core-styles": "tacc-wbomar/Core-Styles.git",
```

Comment:
If we move this to production, we should probably move this to the TACC org

Reply:
Agreed.

_Just putting it here until then. (I didn't want to create it there presumptuously.)_